test/widgets/subscription_list_test.dart

```diff
@@ -89,18 +89,49 @@ void main() {
     check(isUnpinnedHeaderInTree()).isTrue();
   });
 
-  testWidgets('subscription sort', (tester) async {
-    await setupStreamListPage(tester, subscriptions: [
-      eg.subscription(eg.stream(streamId: 1, name: 'd'), pinToTop: true),
-      eg.subscription(eg.stream(streamId: 2, name: 'c'), pinToTop: false),
-      eg.subscription(eg.stream(streamId: 3, name: 'b'), pinToTop: true),
-      eg.subscription(eg.stream(streamId: 4, name: 'a'), pinToTop: false),
-    ]);
-    check(isPinnedHeaderInTree()).isTrue();
-    check(isUnpinnedHeaderInTree()).isTrue();
-
-    final streamListItems = tester.widgetList<SubscriptionItem>(find.byType(SubscriptionItem)).toList();
-    check(streamListItems.map((e) => e.subscription.streamId)).deepEquals([3, 1, 4, 2]);
+  group('subscription sorting', () {
+    streamListIds(WidgetTester tester) => tester
```

Comment:
nit: Write an explicit return type on function definitions:
```suggestion
    Iterable<int> streamListIds(WidgetTester tester) => tester
```

Otherwise the start of the line looks a lot like a function call.